OWNERS

```diff
@@ -1,4 +1,8 @@
 reviewers:
```

Comment:
I'd suggest adding to reviewers:
- zhangxiaoyu-zidif
- xiangpengzhao

They're both active enough that giving them /lgtm power makes sense to me.

Reply:
added!